packages/react/src/blocks/Image.tsx

```diff
@@ -109,11 +109,35 @@ class ImageComponent extends React.Component<any> {
     return getSrcSet(url)
   }
 
+  getSizes(sizes: string) {
```

Comment:
since this doesn't rely on any instance property, what do you think of hoisting it up like the `getSrcSet` function